### ISSUE TITLE
Incorrect Statistics after Submission

### ISSUE THREAD

**Issue by subhajit-rajak** [LeetCode-Feedback/LeetCode-Feedback#15891]
#### Your LeetCode username

subhajitrajak

#### Category of the bug
- [ ] Question - [518. Coin Change II](https://leetcode.com/problems/coin-change-ii/)

#### Description of the bug

This is the problem of coin change
I had submitted this first by recursion gives time error
then by memoization got submitted
then by tabulation got accepted
then for space optimization I had used array to store the previous and current anwer
But in all the cases when I submit the answer the space required and time required is same which should not occur
at the same time when I resubmit the same answer it gives different runtime.

**Comment by LC-Pam** [comment]
The issue you experienced is runtime variation. It normally happens when your code is at the edge of the time limit that can pass the problem. Therefore, your code may have different results at different submissions.
 It’s very difficult and ongoing research to evaluate the program’s runtime and get the same value for different runs of the same program. We have tried our best to make sure we isolate your program’s runtime, but there exist a lot of factors that make different executions have runtime variations.
 
I've passed your feedback to our engineering team and they'll investigate in order to provide consistent results for our users. However, I'm afraid that we won't have a quick fix for the issue that you're facing. I recommend that you optimize your code to pass the time limit. We apologize for the inconvenience and thank you for your support!
